packages/php-wasm/compile/Makefile

```diff
@@ -1,4 +1,30 @@
+# A phony target is one that is not really the name of a file; rather it is
+# just a name for a recipe to be executed when you make an explicit request.
+# There are two reasons to use a phony target: to avoid a conflict with a
+# file of the same name, and to improve performance.
+
+# If you write a rule whose recipe will not create the target file,
+# the recipe will be executed every time the target comes up for remaking.
+# https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
```

Comment:
Generally I love comments like this, but I also think it's okay if we don't add it, because `.PHONY` is a pretty standard feature of Makefile syntax and there should be ample documentation available for someone to understand it. We don't document the other parts of the Makefile syntax here, and it's probably okay to let the PHONY exist as is.